tests/core/ckeditor/manual/hcdetection.md

```diff
@@ -2,7 +2,7 @@
 @bender-ui: collapsed
 @bender-ckeditor-plugins: wysiwygarea, toolbar, undo, clipboard, basicstyles, about, stylescombo, format, link, image
 
-**Note**: this test is dedicated for OSes that **DOES** support high contrast mode (Windows mostly). You can see if HC Mode is correctly detected based on test indicator (`HC is on`).
+**Note**: this test is dedicated for OSes that **DOES** support high contrast mode (Windows mostly). You can see if HC Mode is correctly detected based on test indicator (`HC is on`). This feature is not supported on Google Chrome, Safari and mobile.
```

Comment:
Would be nice to include a link to the docs here: https://ckeditor.com/docs/ckeditor4/latest/guide/skin_sdk_hc.html.